### DISCUSSION TITLE
Add icon for using volume control only

### DISCUSSION THREAD

**Discussion by weensy** [MonitorControl/MonitorControl#996] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

I am a user who only uses the volume control. People who use it like me have a sense of incongruity in function and icon.
<img width="335" alt="Screen Shot 2022-03-26 at 18 11 38" src="https://user-images.githubusercontent.com/54832625/160247505-8badea22-57ef-4d19-a5a7-dc652ee6f641.png">

### Describe the solution you'd like

Prepare icons such as Brightness, Contrast, Sound, and Monitor, and allow users to choose one.

### Describe alternatives you've considered

Alternatively, we can choose a universal icon. It is also discussed in Issues #925.

**Comment by waydabber** [comment]
I understand the sentiment, however changing the icons or even adding custom icon options were voted down several times in the past so I don't dare to even raise the issue. ;)

What you can do is download the source code into XCode and simply rename the icon symbol name to `speaker.wave.2.fill` for example and compile the app, that will give you a nice speaker icon.

<img width="672" alt="Screen Shot 2022-04-07 at 20 01 45" src="https://user-images.githubusercontent.com/37590873/162267612-0b1db9ea-2b6d-4bd4-9b18-f2e79d045d99.png">
